lib/drab/channel.ex

```diff
@@ -70,19 +70,38 @@ defmodule Drab.Channel do
     # for debugging
     if IEx.started? do
       commander = Drab.get_commander(socket)
+
+      assign_example =
+        case Drab.Live.socket_to_assigns(socket) do
+          [] ->
+            nil
+              
+          [first_assign | _] ->
+            first_assign
+        end
+      
       modules = DrabModule.all_modules_for(commander.__drab__().modules)
       grouped = Enum.map(modules, fn module ->
         [_ | rest] = Module.split(module)
         Enum.join(rest, ".")
       end) |> Enum.join(", ")
 
-      module_examples = %{
-        Drab.Live     => "socket |> poke(count: 42)",
-        Drab.Element  => "socket |> set_style(backgroundColor: \"red\")",
+      live_example =
+        # if there are no assigns for the current socket, don't display `Live` example
+        if assign_example do
+          %{Drab.Live => "socket |> poke(#{assign_example}: 42)"}
```

Comment:
Let's change the assign value to a string, like \"forty two\". Some of the assign (eg properties) types may vary, so it is safer to show user a string.